tidy3d/components/data/monitor_data.py

```diff
@@ -2098,6 +2099,78 @@ def renormalize_fields(self, proj_distance: float) -> FieldProjectionAngleData:
         # compute updated fields and their coordinates
         return self.make_renormalized_data(phase, proj_distance)
 
+class DirectivityData(FieldProjectionAngleData):
```

Comment:
Expose this to the tidy3d namespace [here](https://github.com/flexcompute/tidy3d/blob/7f69690abd87b9cf6e76f5b5d0e341ad7b47ba05/tidy3d/__init__.py#L69) and [here](https://github.com/flexcompute/tidy3d/blob/7f69690abd87b9cf6e76f5b5d0e341ad7b47ba05/tidy3d/__init__.py#L254)